protocol.html

```diff
@@ -383,6 +381,21 @@ <h2 about="#introduction" property="schema:name" typeof="deo:Introduction">Intro
                 <li><a href="http://data.europa.eu/esco/occupation/c40a2919-48a9-40ea-b506-1f34f693496d">Application developers</a> who want to implement a client to listen for updates to particular resources.</li>
               </ul>
 
+              <section id="overview" inlist="" rel="schema:hasPart" resource="#overview">
```

Comment:
On one hand I like the Overview closer to the top of the section but it contains terminology that's not clear just yet. That was the initial reason to have it after Terminology.. and thought that the Use Cases section made sense before diving into the technical details..

(As for Use Cases, going forward, that can be truncated and have it rolled into Introduction.. and link to notifications-ucr document.)